ZupIT/ritchie-cli
# Fix- Print error messages in pre builder

### Description

This exposes the errors associated with the pre-build, improving the traceability and visibility of the error content.

Some extra improvements were the addition of new mocks and asserts in the tests.

* In this commit an intermittent test was corrected. -> https://github.com/ZupIT/ritchie-cli/pull/829/commits/972f69b28181a41f793e7ca62dd63f68a5f932cb
* For the names of the mock files I followed the name of the corresponding package

### How to verify it
Check the code or clone this pr, make the build and tests errors on build.

> ![image](https://user-images.githubusercontent.com/67295691/105877154-6d88ea80-5fde-11eb-9605-6ef7124f47dd.png)



### Changelog

Print error message in pre-builder
